catalog/imaging/imaging-actionprovider-chip/src/test/java/org/codice/alliance/imaging/chip/actionprovider/TestImageChipActionProvider.java

```diff
@@ -0,0 +1,139 @@
+
+    private static final String ID = "12345";
+
+    private static final String SOURCE = "alliance.distribution";
+
+    private static final String LOCATION =
+            "POLYGON((0.1234 2.222, 0.4444 1.222, 0.1234 1.222, 0.1234 2.222, 0.1234 2.222))";
+
+    private static final String BAD_WKT_LOCATION = "NOWHERE";
+
+    private ImagingChipActionProvider imagingChipActionProvider;
+
+    private MetacardImpl metacard;
+
+    @Before
+    public void setUp() {
+        imagingChipActionProvider = new ImagingChipActionProvider();
+        metacard = new MetacardImpl();
+        metacard.setContentTypeName(ImagingChipActionProvider.NITF_CONTENT_TYPE);
+        metacard.setId(ID);
+        metacard.setSourceId(SOURCE);
+        metacard.setLocation(LOCATION);
+    }
+
+    @Test
+    public void testCanHandleNullMetacard() {
+        assertThat(imagingChipActionProvider.canHandle(null), is(false));
+    }
+
+    @Test
+    public void testCanHandleInvalidMetacard() {
+        assertThat(imagingChipActionProvider.canHandle(new Object()), is(false));
+    }
+
+    @Test
+    public void testCanHandleInvalidLocationOnMetacard() {
+        metacard.setLocation(null);
+        assertThat(imagingChipActionProvider.canHandle(metacard), is(false));
+        metacard.setLocation(LOCATION);
+    }
+
+    @Test
+    public void testCanHandleNullContentTypeMetacard() {
+        MetacardImpl metacard = new MetacardImpl();
+        assertThat(imagingChipActionProvider.canHandle(metacard), is(false));
+    }
+
+    @Test
+    public void testCanHandleNonImageryMetacard() {
+        MetacardImpl metacard = new MetacardImpl();
+        metacard.setContentTypeName(ZIP_CONTENT_TYPE);
+        assertThat(imagingChipActionProvider.canHandle(metacard), is(false));
+    }
+
+    @Test
+    public void testCanHandleImageryMetacard() {
+        assertThat(imagingChipActionProvider.canHandle(metacard), is(true));
+    }
+
+    @Test
+    public void testGetId() {
+        assertThat(imagingChipActionProvider.getId(), is(ImagingChipActionProvider.ID));
+    }
+
+    @Test
+    public void testGetActionsNullMetacard() {
+        assertThat(imagingChipActionProvider.getActions(null), nullValue());
+    }
+
+    @Test
+    public void testGetActionsUnsupportedProtocolOnMetacard() {
+        String protocol = SystemBaseUrl.getProtocol();
+        System.setProperty("org.codice.ddf.system.protocol", "udp://");
+        assertThat(imagingChipActionProvider.getActions(metacard), nullValue());
+        System.setProperty("org.codice.ddf.system.protocol", protocol);
+    }
+
+    @Test
+    public void testGetActionsCanNotHandleMetacard() {
+        assertThat(imagingChipActionProvider.getActions(new Object()), nullValue());
+    }
+
+    @Test
+    public void testGetActionsBadLocationMetacard() {
+        metacard.setLocation(BAD_WKT_LOCATION);
+        assertThat(imagingChipActionProvider.getActions(metacard), nullValue());
+        metacard.setLocation(LOCATION);
```

Comment:
Your `setUp()` method is making a new metacard for every test, so you don't need to reset it.